<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(197, 202, 233);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>40</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>48</red>
          <green>63</green>
          <blue>159</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>48</red>
          <green>63</green>
          <blue>159</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>48</red>
          <green>63</green>
          <blue>159</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>48</red>
          <green>63</green>
          <blue>159</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>48</red>
          <green>63</green>
          <blue>159</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>48</red>
          <green>63</green>
          <blue>159</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>190</red>
          <green>190</green>
          <blue>190</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>48</red>
          <green>63</green>
          <blue>159</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>48</red>
          <green>63</green>
          <blue>159</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>48</red>
          <green>63</green>
          <blue>159</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(48, 63, 159);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>IV Monitor</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>781</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="title">
        <string>Device5</string>
       </property>
       <widget class="QLCDNumber" name="d5v">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>161</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
       </widget>
       <widget class="QLCDNumber" name="d5min">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>110</y>
          <width>64</width>
          <height>23</height>
         </rect>
        </property>
       </widget>
       <widget class="QLCDNumber" name="d5max">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>110</y>
          <width>64</width>
          <height>23</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>59</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>Min</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>90</y>
          <width>59</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>Max</string>
        </property>
       </widget>
       <widget class="QLabel" name="statusd5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>150</y>
          <width>171</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>status</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_6">
       <property name="title">
        <string>Device6</string>
       </property>
       <widget class="QLCDNumber" name="d6v">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>161</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
       </widget>
       <widget class="QLCDNumber" name="d6min">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>110</y>
          <width>64</width>
          <height>23</height>
         </rect>
        </property>
       </widget>
       <widget class="QLCDNumber" name="d6max">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>110</y>
          <width>64</width>
          <height>23</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_17">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>59</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>Min</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_18">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>90</y>
          <width>59</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>Max</string>
        </property>
       </widget>
       <widget class="QLabel" name="statusd6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>150</y>
          <width>171</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>status</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_7">
       <property name="title">
        <string>Device7</string>
       </property>
       <widget class="QLCDNumber" name="d7v">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>161</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
       </widget>
       <widget class="QLCDNumber" name="d7min">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>110</y>
          <width>64</width>
          <height>23</height>
         </rect>
        </property>
       </widget>
       <widget class="QLCDNumber" name="d7max">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>110</y>
          <width>64</width>
          <height>23</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_20">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>59</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>Min</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_21">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>90</y>
          <width>59</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>Max</string>
        </property>
       </widget>
       <widget class="QLabel" name="statusd7">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>150</y>
          <width>171</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>status</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_8">
       <property name="title">
        <string>Device8</string>
       </property>
       <widget class="QLCDNumber" name="d8v">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>161</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
       </widget>
       <widget class="QLCDNumber" name="d8min">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>110</y>
          <width>64</width>
          <height>23</height>
         </rect>
        </property>
       </widget>
       <widget class="QLCDNumber" name="d8max">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>110</y>
          <width>64</width>
          <height>23</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_23">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>59</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>Min</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_24">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>90</y>
          <width>59</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>Max</string>
        </property>
       </widget>
       <widget class="QLabel" name="statusd8">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>150</y>
          <width>171</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>status</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>679</x>
      <y>430</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>SETTING</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>781</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Device1</string>
       </property>
       <widget class="QLCDNumber" name="d1v">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>161</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
       </widget>
       <widget class="QLCDNumber" name="d1min">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>110</y>
          <width>64</width>
          <height>23</height>
         </rect>
        </property>
       </widget>
       <widget class="QLCDNumber" name="d1max">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>110</y>
          <width>64</width>
          <height>23</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>59</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>Min</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>90</y>
          <width>59</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>Max</string>
        </property>
       </widget>
       <widget class="QLabel" name="statusd1">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>150</y>
          <width>171</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>status</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Device2</string>
       </property>
       <widget class="QLCDNumber" name="d2v">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>161</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
       </widget>
       <widget class="QLCDNumber" name="d2min">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>110</y>
          <width>64</width>
          <height>23</height>
         </rect>
        </property>
       </widget>
       <widget class="QLCDNumber" name="d2max">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>110</y>
          <width>64</width>
          <height>23</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>59</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>Min</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>90</y>
          <width>59</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>Max</string>
        </property>
       </widget>
       <widget class="QLabel" name="statusd2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>150</y>
          <width>171</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>status</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Device3</string>
       </property>
       <widget class="QLCDNumber" name="d3v">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>161</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
       </widget>
       <widget class="QLCDNumber" name="d3min">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>110</y>
          <width>64</width>
          <height>23</height>
         </rect>
        </property>
       </widget>
       <widget class="QLCDNumber" name="d3max">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>110</y>
          <width>64</width>
          <height>23</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>59</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>Min</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>90</y>
          <width>59</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>Max</string>
        </property>
       </widget>
       <widget class="QLabel" name="statusd3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>150</y>
          <width>171</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>status</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string>Device4</string>
       </property>
       <widget class="QLCDNumber" name="d4v">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>161</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
       </widget>
       <widget class="QLCDNumber" name="d4min">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>110</y>
          <width>64</width>
          <height>23</height>
         </rect>
        </property>
       </widget>
       <widget class="QLCDNumber" name="d4max">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>110</y>
          <width>64</width>
          <height>23</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>59</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>Min</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>90</y>
          <width>59</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>Max</string>
        </property>
       </widget>
       <widget class="QLabel" name="statusd4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>150</y>
          <width>171</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>status</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
